core/logging_api.php

```diff
@@ -110,12 +112,11 @@ function log_event( $p_level, $p_msg ) {
 	if( function_exists( 'event_signal' ) ) {
 		event_signal( 'EVENT_LOG', array( $t_plugin_event ) );
 	}
-
```

Comment:
Please don't apply formatting changes.  Also the formatting changes applied are not inline with coding style for MantisBT.